timed/employment/views.py

```diff
@@ -174,7 +174,7 @@ def _extract_date(self):
                 return datetime.datetime.strptime(pk.split("_")[1], "%Y-%m-%d")
 
             except (ValueError, TypeError, IndexError):
-                raise exceptions.NotFound()
+                raise exceptions.NotFound from None
```

Comment:
Shouldn't these raise from the original exception? It feels like we are throwing away readily available info here.

```python
            except (ValueError, TypeError, IndexError) as ex:
                raise exceptions.NotFound from ex
```

Reply:
i'm not sure

Reply:
i've adjusted it now